docs/tools/aiven-console.rst

```diff
@@ -1,71 +1,59 @@
 - :doc:`Managing two-factor authentication </docs/platform/howto/user-2fa>`
 - :doc:`Generating authentication tokens </docs/platform/concepts/authentication-tokens>`
 
-Payment cards
-"""""""""""""
-In the **Payment Options** section, you can :doc:`add or update credit card details </docs/platform/howto/manage-payment-card>`, assign projects to a payment method, and update the :doc:`billing contact </docs/platform/howto/change-billing-contact>`.
-
 Organization and organizational unit settings
 ----------------------------------------------
 
-The organization or organizational unit that you are currently working with is displayed at the top of the page. You can switch to another organization or organizational unit by clicking the name to open the drop-down menu. You can also create a new organization from this menu.
+The :doc:`organization or organizational unit </docs/platform/concepts/projects_accounts_access>` that you are currently working with is displayed at the top of the page. You can switch to another organization or organizational unit by clicking the name to open the drop-down menu. 
 
 If you don't have an organization, click **Create organization** to :doc:`create your first organization</docs/tools/aiven-console/howto/create-accounts>`. 
  
 .. note:: We strongly recommend creating an organization. It makes managing your projects much easier and comes with many additional features, such as teams (user groups), billing groups, and SAML authentication.
 
 Organization and organizational unit settings are available on the **Admin** page. Here you can:
 
-* :doc:`Manage the teams</docs/tools/aiven-console/howto/create-manage-teams>` 
+* :doc:`Manage your teams</docs/tools/aiven-console/howto/create-manage-teams>` 
 * Create new projects under an organization or organizational unit
 * Configure :doc:`authentication methods for an organization </docs/platform/howto/list-saml>`
-* View **Event logs** of activity such as the adding or removing of team members, changing authentication methods, and more
-* Access the **Settings** to rename or delete an organization or organizational unit 
-
+* View logs of activity such as the adding or removing of team members, changing authentication methods, and more
+* Rename or delete an organization or organizational unit 
 
 Projects and services
 ----------------------
 
-To navigate between different projects or view all projects click the **Projects** drop-down menu. Selecting a project opens the **Current services** page with a list of all services for that project. Here you see a list of all of your services and :doc:`create services </docs/platform/howto/create_new_service>`.
+To navigate between different projects or view all projects click the **Projects** drop-down menu. Selecting a project opens the **Services** page with a list of all services in that project. Here you can view the status of the services and :doc:`create new services </docs/platform/howto/create_new_service>`.
```

Comment:
We should add a note here which mentions that only projects which belong to the current Org / Unit are displayed. So it's not all projects you have access to.